polaris.shopify.com/content/components/navigation.md

```diff
@@ -62,6 +62,7 @@ The navigation component should:
 - Use a major icon for item actions.
 - Use a minor icon for secondary actions.
 - Use the provided navigation section component to group navigation items.
+- Not use the [New badge](/patterns/new-badge) in navigation items.
```

Comment:
I wonder if the documentation could be.. "Do not add additional components like `Badge` into the navigation". I foresee people trying to sneak other components in.

Reply:
I like that suggestion because the clarity future proofs potential misuse/interpretations. Suggesting this copy in the [PR](https://github.com/Shopify/polaris/pull/8118) so that people can find the badge component and our updated new badge pattern guidelines: Not add additional components, like [badge](/components/badge), in navigation items. Example: Don‘t add the [New badge](/patterns/new-badge).